README.md

```diff
@@ -1,2 +1,39 @@
 # LePetitBloc.github.io
+
 LePetitBloc Website
+
+## Code of conduct
+
+See the [CODE OF CONDUCT](CODE_OF_CONDUCT.md) file.
+
+## Contributing
+
+See the [CONTRIBUTING](CONTRIBUTING.md) file.
+
+## License
+
+This project is under [MIT License](LICENSE.md)
+
+## Support
+
+See the [SUPPORT](SUPPORT.md) file.
+
+## Credits
```

Comment:
These specific credits could be kept in the `repository-template` repository. 
@pocky what do you think? I personally think they aren't relevant here.